### ISSUE TITLE
Timeslider disappearing with individual settings

### ISSUE THREAD

**Issue by dpxgit** [mediaelement/mediaelement-plugins#207]
When you manually set background-color to black and foreground-color to white in your browser, the horizontal-volume-slider becomes invisible. This affects accessibility and could be solved with border-colors.

![image](https://user-images.githubusercontent.com/80531038/135613806-b6d488a7-4460-4522-a753-4c19302bd5a7.png)
